docs/pages/admin-guides/teleport-policy/integrations/crown-jewels.mdx

```diff
@@ -0,0 +1,99 @@
+
+## Prerequisites
+
+- A running Teleport Enterprise cluster v16.2.0 or later.
+- For self-hosted clusters, an updated `license.pem` with Teleport Policy enabled.
+- For self-hosted clusters, a running Access Graph node v1.24.0 or later.
+Check [Access Graph page](../teleport-policy.mdx) for details on
+how to set up Access Graph.
+
+Access Graph is a feature of the [Teleport Policy](https://goteleport.com/platform/policy/) product
+available to Teleport Enterprise edition customers.
+
+After logging in to the Teleport UI, navigate to the Management tab. If enabled, Access Graph options can be found
+under the Permission Management section.
+
+## Creating a Crown Jewel
+
+To create a Crown Jewel, you need to mark a resource or user as critical. Only changes to marked resources
+and users will be logged by Teleport Policy.
+To mark a resource or user as Crown Jewel, open the Access Graph and navigate to the "Crown Jewels" tab.
+
+![Create Page](../../../../img/access-graph/crown-jewels/create-page.webp)
+
+
+Click on "Create Crown Jewel" and select the resource or user you want to mark as critical.
+
+![Create Selector](../../../../img/access-graph/crown-jewels/create-selector.webp)
+
+Pick a name for the Crown Jewel and click "Create".
+
+![Create Matcher Name](../../../../img/access-graph/crown-jewels/create-matcher-name.webp)
+
+The Crown Jewel will now be created, and you will see it in the list of Crown Jewels. Access Graph will now create
+audit events in Teleport's audit log and new entries in the "Access Changes" tab in the "Crown Jewels" menu whenever
+access path to a resource or a user changes.
+
+## Viewing permission changes
+
+To view permission changes, open the Access Graph and navigate to the "Crown Jewels" tab.
+Here you can see a list of all Crown Jewels and the changes that have been made to them.
+
+![Changes](../../../../img/access-graph/crown-jewels/changes.webp)
+
+The generated changes have a diff format showing removed nodes with "-" and added nodes with "+".
+Every time a change is made to a Crown Jewel, a new entry will be added to the list and an audit event
+will be generated in the Teleport Audit Log.
+
+## Audit events
+
+Every time a change is made to a Crown Jewel, an audit event is generated in the Teleport Audit Log.
+Here is what an example audit event looks like:
+
+```json
+{
+  "affected_resource_name": "bob",
+  "affected_resource_source": "TELEPORT",
+  "affected_resource_type": "teleport_node",
+  "change_id": "0110b3c4-d0b5-4af9-8585-aa49a064c85d",
+  "cluster_name": "ssh-node",
+  "code": "TAG001I",
+  "ei": 0,
+  "event": "access_graph.access_path_changed",
+  "time": "2024-09-20T19:50:38.194Z",
+  "uid": "5447d050-699a-4009-a901-ab8ed2614bc2"
+}
+```
+
+You can export the audit event using the event handler.
+The setup is described [here](../../management/export-audit-events.mdx).
+
+## Required RBAC permissions
```

Comment:
This seems more like a prerequisite, doesn't it? If you don't have the required permissions you won't be able to do any of the actions described in this guide.

Reply:
In this case we should include the required RBAC role for every feature which we're currently not doing in any guide. I added it here, so people have an example role that can be added to an existing users. New TAG version also displays which RBAC permissions are required to use a feature when a user doesn't have them.

Reply:
Can we at least move it earlier in the guide?

It's a bit awkward to explain all the details about Crown Jewels, and then at the very end after the reader has made their way through the whole page we say "by the way make sure you have these permissions"

Reply:
Sure, done.